### ISSUE TITLE
Question about encoding of `<supplied>` together with `<gap>`

### ISSUE THREAD

**Issue by ajaniak** [erc-dharma/tfd-nusantara-philology#215]
Dear @arlogriffiths,

I am working on updating the XSLT for <gap/>. I have a question regarding the processing of `<supplied reason="lost" evidence="parallel"><gap/></supplied>`
The current display is  [[…]]

See several cases:

<img width="1045" alt="Image" src="https://github.com/user-attachments/assets/63818534-1032-4bf9-bc5f-5738ac2877d8" />

I wanted to know if it was ok or if you only wanted one piece of brackets before taking time on this matter. Still might not be an emergency

Thanks, 
Best, 
Axelle

**Comment by arlogriffiths** [comment]
Do you know this page https://dharmalekha.info/editorial-conventions?

Whenever possible, let's try to follow the conventions defined there.

This one, concerning a locus where we know based on parallels that there must a gap in the transmitted text but we cannot propose a way to fill it in, does not appear in that list (but maybe @danbalogh or @michaelnmmeyer  could add it). Display with [[...]] seems consistent with the set of conventions as a whole.

**Comment by danbalogh** [comment]
I don't understand how a gap (i.e. a lacuna) can be lost in the first place, as loss = gap; I understand even less how you could supply a gap. Is the intention rather `<supplied reason="lost" evidence="parallel"><space type="vacat"/></supplied>`? Or what am I missing here? Is it the case that the lacuna was only illegible to the previous editor but it is completely lost now?

**Comment by ajaniak** [comment]
I have asked the question because I could find this equivalent in the cases listed https://dharmalekha.info/editorial-conventions?

**Comment by arlogriffiths** [comment]
Thanks Dan and Axelle.

@Dan: No, we don't mean vacat but a suspected locus of textual loss of unknown extent that cannot be restored. Remember that this concerns a critical edition, based on one or more manuscripts. The scenario is discussed in EGC 5.7.1 (Reporting a locus of suspected textual loss):

"A particular scenario arises when you have reason to believe, against unanimous evidence of the manuscripts, that a certain amount of text has been omitted or lost in transmission at a given locus, but are unable to determine what precise segment of text is to be supplied to fill the gap. This is basically the opposite of an interpolation. [...] You may also use "lost" or "unknown" as values of @reason (in many cases, it will be impossible to know the reason for a segment’s absence). Use of @evidence is not mandatory ..."

The encoding with `<supplied reason="omitted" evidence="parallel"><gap/></supplied>` is proposed by the guide and it seems adequate to me. Please let us know if you have a better idea.

**Comment by danbalogh** [comment]
Thanks, I think I understand the situation now. This reminds me very much of EGD §6.4 (in the [Google Doc](https://docs.google.com/document/d/1hjWrrwRZQp4hmEqw4jBhhqoXdwJvRlw3EWboJteOPw0/edit?tab=t.0#heading=h.yh4i4spcmjh)), "Scribal Omission without Editorial Restoration". The only difference seems to be that in the EGD case the unanimous evidence of the manuscripts is of course limited to a single manuscript. The encoding is `<gap reason="omitted" unit="character"/>` with extent/quantity as applicable (i.e. including `@extent="unknown"` when applicable).
I must note that I'm also not very happy with the EGD solution (there must have been some discussion of this previous to the addition of this section in January 2021, but I don't recall it), but I still find it semantically more appropriate that the notion that the editor supplies - i.e. adds to the text - a lacuna - i.e. something missing from the text.
At any rate, I have no serious attachment to any solution; if you want to stick to gap within supplied, that is also acceptable to me, though I would definitely say that the reason should in that case be "omitted" and not "lost", which means physical loss of text that had once been inscribed on the support.
